weave-js/src/components/PagePanelComponents/Home/Browse3/StyledDataGrid.tsx

```diff
@@ -77,5 +79,11 @@ export const StyledDataGrid = styled(
         backgroundColor: backgroundColorHoveredSelected,
       },
     },
+    '&.bg-red-100': {
```

Comment:
nit: Using `bg-red-100` implies that you are using some `red-100`, but i am seeing a more complicated color here. Since we are codifying the rules for "deletion styling", i would recommend changing this to `selected-for-deletion` classname. Then making that a constant that is used here and in the CallsTable. This way the class describes the state, rather than the implementation of the state

Reply:
ack yeah thats a mistake, good idea...